lib/generators/graphiti/install_generator.rb

```diff
@@ -12,17 +12,32 @@ class InstallGenerator < ::Rails::Generators::Base
       aliases: ["-c"],
       desc: "Generate without documentation comments"
 
+    class_option :'namespace-controllers',
+      type: :boolean,
+      default: false,
+      aliases: ["-n"],
+      desc: "Generated controllers will be namespaced"
+
     desc "This generator boostraps graphiti"
     def install
       to = File.join("app/resources", "application_resource.rb")
       template("application_resource.rb.erb", to)
 
-      inject_into_file "app/controllers/application_controller.rb", after: "class ApplicationController < ActionController::API\n" do
-        app_controller_code
-      end
+      if namespace_controllers?
+        to = File.join(
+          "app/controllers",
+          (controller_namespaces_path if namespace_controllers?),
```

Comment:
@LoganDSPrice will `namespace_controllers?` always be true in this case? If so, is this conditional required?

Reply:
When you run `rails generate graphiti:install`, by default `namespace-controllers` in `graphiticfg.yml` is set to `false`.  Only by passing `--namespace-controllers` will it be set to `true`.

Reply:
That makes sense. I think I worded my question poorly before. In the case where on line 26, `namespace_controllers?` _is_ `true` and then we get to line 29, won't `namespace_controllers?` still always be `true`? You checked the status of `namespace_controllers?` once, do you need to do it again here? In the case where `namespace_controllers?` is  `false` is handled by the `else` clause below right?